x-pack/plugins/fleet/server/services/index.ts

```diff
@@ -35,6 +35,7 @@ export interface ESIndexPatternService {
 
 export interface PackageService {
   getInstallation: typeof getInstallation;
+  getPackageInfo: typeof getPackageInfo;
```

Comment:
Maybe I missed something but looks like it's not used anywhere no?

Reply:
you're right, I added it on my code, but removed it later and forgot to remove it. Thanks, I'll remove it.